controller/scheduler-v2/scheduler_test.go

```diff
@@ -282,13 +282,13 @@ func (TestSuite) TestExponentialBackoffNoHosts(c *C) {
 			c.Fatalf("expected JobRequestEvent, got %T", event)
 		}
 		c.Assert(e.Request, NotNil)
-		c.Assert(e.Request.restarts, Equals, expected)
 	}
 
 	// wait for the formation to cascade to the scheduler
 	s.waitRectify()
-	waitRestarts(1*time.Second+50*time.Millisecond, 2)
-	waitRestarts(2*time.Second+50*time.Millisecond, 3)
```

Comment:
Switch to linear restart attempts for job requests (exponential for crashed job restarts)